projects/plugins/jetpack/extensions/plugins/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Internal dependencies
+ */
+import registerJetpackPlugin from '../shared/register-jetpack-plugin';
+
+/*
+ * Jetpack core-editor plugins dependencies
+ */
+import { name as publicizePluginName, settings as publicizePluginSettings } from './publicize';
+import { name as sharingPluginName, settings as sharingPluginSettings } from './sharing';
+
+/*
+ * Register plugins.
+ * Order matters. The plugins will be registered in the order they are listed here,
+ * impacting the order in which they are rendered in the UI.
+ *
+ * https://github.com/Automattic/jetpack/issues/21036
+ */
+registerJetpackPlugin( publicizePluginName, publicizePluginSettings );
+registerJetpackPlugin( sharingPluginName, sharingPluginSettings );
```

Comment:
This may get to be really repetitive as we expand that list. Would it be possible to instead have a single array we'd loop through?

Reply:
Addresses here 0bfbbfb0f545e5a15fb1708abe4d303299d52299